ui/mainwindow.py

```diff
@@ -48,14 +49,19 @@ def setupUi(self, MainWindow):
         self.tabWidget.addTab(self.tab, "")
         self.gridLayout_3.addWidget(self.tabWidget, 0, 0, 1, 1)
         self.sceneWidget = QtWidgets.QWidget(self.centralwidget)
+        self.instrumentView = InstrumentView()
         sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
         sizePolicy.setHorizontalStretch(1)
         sizePolicy.setVerticalStretch(0)
         sizePolicy.setHeightForWidth(self.sceneWidget.sizePolicy().hasHeightForWidth())
         self.sceneWidget.setSizePolicy(sizePolicy)
         self.sceneWidget.setMinimumSize(QtCore.QSize(745, 0))
         self.sceneWidget.setObjectName("sceneWidget")
-        self.gridLayout_3.addWidget(self.sceneWidget, 0, 1, 1, 2)
+        self.instrumentView.setSizePolicy(sizePolicy)
+        self.instrumentView.setMinimumSize(QtCore.QSize(745, 0))
+        self.instrumentView.setObjectName("instrumentView")
+        # self.gridLayout_3.addWidget(self.sceneWidget, 0, 1, 1, 2)
```

Comment:
you need to make these changes in mainwindow.py in the `nexus_constructor/` directory, these will be removed when the ui file gets changed and `pyside2-uic` is used. The one in `nexus_constructor/` is the actual implementation.